app/routes/titiler/algorithms/alerts.py

```diff
@@ -9,31 +9,29 @@
 
 from app.models.enumerators.alerts_confidence import DeforestationAlertConfidence
 
-START_DATE: str = "2014-12-31"  # start of record
-
-ALERT_CONFIDENCE_MAP: OrderedDict = OrderedDict(
-    {
-        DeforestationAlertConfidence.low: {
-            "confidence": 2,
-            "colors": {"red": 237, "green": 164, "blue": 194},
-        },
-        DeforestationAlertConfidence.high: {
-            "confidence": 3,
-            "colors": {"red": 220, "green": 102, "blue": 153},
-        },
-        DeforestationAlertConfidence.highest: {
-            "confidence": 4,
-            "colors": {"red": 201, "green": 42, "blue": 109},
-        },
-    }
-)
-
-
-class IntegratedAlerts(BaseAlgorithm):
-    """Decode Integrated Alerts."""
-
-    title: str = "Integrated Deforestation Alerts"
+
+class Alerts(BaseAlgorithm):
+    """Decode Deforestation Alerts."""
+
+    title: str = "Deforestation Alerts"
     description: str = "Decode and visualize alerts"
+    conf_colors: OrderedDict = OrderedDict(
+        {
+            DeforestationAlertConfidence.low: {
+                "confidence": 2,
+                "colors": (237, 164, 194),
+            },
+            DeforestationAlertConfidence.high: {
+                "confidence": 3,
+                "colors": (220, 102, 153),
+            },
+            DeforestationAlertConfidence.highest: {
+                "confidence": 4,
+                "colors": (201, 42, 109),
+            },
+        }
+    )
```

Comment:
I ❤️ going with the immutable tuple here, @solomon-negusse.

I also like the descriptiveness of the _red, green, blue_ keys of the original `dict`. 

Did you consider using a namedTuple to get the best of both worlds?
Something like:
```python
# Define a namedtuple for Colors (immutable)
Colors = namedtuple('Colors', ['red', 'green', 'blue'])

# Define a namedtuple for DeforestationAlert (immutable)
DeforestationAlert = namedtuple('DeforestationAlert', ['confidence', 'colors'])

# Assuming DeforestationAlertConfidence is an Enum or similar object
conf_colors: OrderedDict = OrderedDict(
    {
        DeforestationAlertConfidence.low: DeforestationAlert(confidence=2, colors=Colors(237, 164, 194)),
        DeforestationAlertConfidence.high: DeforestationAlert(confidence=3, colors=Colors(220, 102, 153)),
        DeforestationAlertConfidence.highest: DeforestationAlert(confidence=4, colors=Colors(201, 42, 109)),
    }
)
```

Reply:
Great idea, thanks!